servo/connectors/kubernetes.py

```diff
@@ -875,10 +876,18 @@ async def create(self, namespace: str = None) -> None:
         if namespace is None:
             namespace = self.namespace
 
-        self.logger.info(f'creating pod "{self.name}" in namespace "{self.namespace}"')
+        self.logger.info(f'Attempting to create a pod named {self.name} in the namespace {self.namespace}.')
         self.logger.trace(f"pod: {self.obj}")
 
         async with self.preferred_client() as api_client:
+            async with kubernetes_asyncio.watch.Watch().stream(api_client.list_pod_for_all_namespaces) as stream:
```

Comment:
Since this is my first attempt at using the Kubernetes API, I've taken what I am sure is a very naieve approach to grabbing pod data. I'm sure this is subscribing us to all sorts of data that we don't want or need. If this is fine for the time being, I can turn this from a draft request into an actual PR.